### DISCUSSION TITLE
Add an option to add map to favorites in song select

### DISCUSSION THREAD

**Discussion by IVII1** [ppy/osu#25370] [Ideas] [answered]
Sometimes i like the map but forget to add it to favorites, so this would be cool. You can add this option in the right click menu. Only applies to maps with at least one difficulty passed so it doesnt change from stable.

**Comment by Joehuu** [comment, marked as answer]
Duplicate of https://github.com/ppy/osu/discussions/21928. Please upvote the original discussion!
